pkgs/applications/misc/prusa-slicer/super-slicer.nix

```
@@ -1,4 +1,4 @@
-{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer }:
+{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer, wxGTK31-gtk3, libspnav }:
```

```suggestion
{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer, libspnav }:
```
This should be enough to fix the eval issues